### PR TITLE
Update branca to v0.4.0 shape

### PR DESCRIPTION
**Pull request by grempe**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://github.com/tuupola/branca-js/commit/64ae114c8942e4700b0f37f587b760a29e99ec85
- [x] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

### CODE REVIEW

File: types/branca/branca-tests.ts [hunk 1]
```diff
@@ -1,4 +1,4 @@
-const key = 'supersecretkeyyoushouldnotcommit';
+const key = '7ed049e344f73f399ba1f7868cf9494f4b13347ecce02a8e463feb32507b73a5';
```
Comment (samarmohan) [inline types/branca/branca-tests.ts:1]: Why did you commit the key?

Comment (grempe) [reply types/branca/branca-tests.ts:1]: This is not a *real* key, its a sample 32 byte hex string, the same sample string used in the upstream repository.

You cannot use just any string as the key (as was used previously). It *must* be a 32 byte (64 char) hex string, or a 32 byte `Buffer`. If it doesn't match this shape the init will fail.

Comment (grempe) [reply types/branca/branca-tests.ts:1]: See the sample usage of the same key in the project README.md

https://github.com/tuupola/branca-js#secret-key

Comment (samarmohan) [reply types/branca/branca-tests.ts:1]: Oops my fault, sorry!

**Review by samarmohan — APPROVED** [review]

**Comment by samarmohan** [comment]
Ready to merge

**Comment by grempe** [comment]
Ready to merge

File: types/branca/branca-tests.ts (hunk 1 above)
Comment (grempe) [reply types/branca/branca-tests.ts:1]: Thanks for merging. Cheers.

**Pull request merged**